variants/aslcanx2/pins_arduino.h

```diff
@@ -0,0 +1,74 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+#include "soc/soc_caps.h"
+
+#define USB_VID 0x303a
+#define USB_PID 0x1001
+
+#define EXTERNAL_NUM_INTERRUPTS 46
+#define NUM_DIGITAL_PINS        48
+#define NUM_ANALOG_INPUTS       20
```

Comment:
Those defines were moved from variants and are defined globally per SoC. Please remove it to fix warnings.
```suggestion
```

Reply:
thanks, I manually applied this change.